app/src/main/java/org/wikipedia/editactionfeed/AddTitleDescriptionsActivity.kt

```diff
@@ -31,25 +32,27 @@ class AddTitleDescriptionsActivity : SingleFragmentActivity<AddTitleDescriptions
     }
 
     override fun onCreateOptionsMenu(menu: Menu): Boolean {
-        menuInflater.inflate(R.menu.menu_edit_action_feed, menu)
+        menuInflater.inflate(R.menu.menu_edit_tasks, menu)
         return true
     }
 
     override fun onOptionsItemSelected(item: MenuItem): Boolean {
         return when (item.itemId) {
             android.R.id.home -> {
                 NavUtils.navigateUpFromSameTask(this)
+                if (WikipediaApp.getInstance().haveMainActivity()) {
+                    NavUtils.navigateUpFromSameTask(this)
+                } else {
+                    startActivity(EditTasksActivity.newIntent(this))
+                }
+                finish()
                 return true
             }
 
             R.id.menu_help -> {
                 FeedbackUtil.showAndroidAppEditingFAQ(baseContext)
                 true
             }
-            R.id.menu_my_contributions -> {
-                // TODO: go to My contributions
-                true
-            }
```

Comment:
Are we sure this must be removed? If I remember right we need to be able to access contributions from task list screen, Add Descriptions overflow and Add Translations overflow..

Reply:
Thanks! Not sure why I removed it, but I add it back now 👍